docs/docusaurus/docs/core/installation_and_setup/manage_credentials.md

```diff
@@ -1,145 +1,214 @@
+To access a deployment environment or a data storage system you must provide your access credentials. These access credentials must be stored securely outside of version control. With GX 1.0 you can store you access credentials as environment variables, in a YAML file that exists outside of version control, or in a third-party secrets manager. GX 1.0 supports the AWS Secrets Manager, Google Cloud Secret Manager, and Azure Key Vault secrets managers. 
 
-<Tabs 
-  queryString="credential-style" 
-  groupId="credentials" 
-  defaultValue='yaml' 
-  values={
-   [
-    {label: 'Environment variables', value:'environment_variables'},
-    {label: 'YAML file', value:'yaml'},
-    {label: 'Secrets manager', value:'secrets_manager'},
-   ]
-  }>
+## Environment variables
 
-  <TabItem value="yaml" label="YAML file">
-  </TabItem>
+1. To store your database password and connection string as environment variables, run ``export ENV_VAR_NAME=env_var_value`` in a terminal or add the commands to your ``~/.bashrc`` file. For example:
 
-  <TabItem value="environment_variables" label="Environment variables">
-  </TabItem>
+    ```bash title="Terminal" name="docs/docusaurus/docs/oss/guides/setup/configuring_data_contexts/how_to_configure_credentials.py export_env_vars"
+    ```
 
-  <TabItem value="secrets_manager" label="Secrets manager">
+2. Run the following code to use the `connection_string` parameter values when you add a `datasource` to a Data Context:
 
-Select one of the following secret manager applications:
+    ```python title="Python" name="docs/docusaurus/docs/oss/guides/setup/configuring_data_contexts/how_to_configure_credentials.py add_credentials_as_connection_string"
+    ```
 
-  <Tabs queryString="manager" groupId="manager" defaultValue='aws' values={[{label: 'AWS Secrets Manager', value:'aws'}, {label: 'GCP Secret Manager', value:'gcp'}, {label: 'Azure Key Vault', value:'azure'}]}>
+## YAML file
 
-  <TabItem value="aws">
-  </TabItem>
+YAML files make variables more visible, are easier to edit, and allow for modularization. For example, you can create a YAML file for development and testing and another for production.
 
-  <TabItem value="gcp">
-  </TabItem>
+The default ``config_variables.yml`` file located at ``great_expectations/uncommitted/config_variables.yml`` applies to deployments using ``FileSystemDataContexts``.
 
-  <TabItem value="azure">
-  </TabItem>
+To view all the code used in the examples, see [how_to_configure_credentials.py](https://github.com/great-expectations/great_expectations/tree/develop/docs/docusaurus/docs/oss/guides/setup/configuring_data_contexts/how_to_configure_credentials.py).
```

Comment:
Rather than link out to a file on GitHub (which will also show all of the `# <snippet>` tags and extra clutter, I think it would be better if this section was split into a "Procedure" tab with the current procedure and a "Sample code" tab that shows the full sample code inside a code block within the document itself.